### DISCUSSION TITLE
Biomes/Map areas

### DISCUSSION THREAD

**Discussion by julianogales** [SIGGgames/PacoMonkey#21] [Ideas]
1. **Tropical**
2. **Forest**
3. **Rocky Mountain**
4. **Snowy Mountain**
5. **Island**
6. **Volcano Mountain** _(inside Island)_

Extra biome (_easter egg_): **Zoo**

**Comment by Aniol0012** [comment]
# Biomes/Map areas preview
This are some concept ideas made with [Chat GPT 4](https://chat.openai.com/):

## 1. **Tropical**:
![topical_biome](https://github.com/SIGGgames/PacoMonkey/assets/53788631/2144d9a1-c0c8-46c8-96c3-31048270d2c4)
## 2. **Forest**:
![forest_biome](https://github.com/SIGGgames/PacoMonkey/assets/53788631/fcd4fa0d-3c4a-454d-a992-2ac44f303aa9)
## 3. **Rocky Mountain**:
![rocky_mountain_biome](https://github.com/SIGGgames/PacoMonkey/assets/53788631/d2d139fd-0579-4910-b7d7-805d7d5142bf)
## 4. **Snowy Mountain**:
![snowy_mountain_biome](https://github.com/SIGGgames/PacoMonkey/assets/53788631/7a63e4a5-8257-4210-8cc0-7553233e9d50)
## 5. **Island**:
![island_biome](https://github.com/SIGGgames/PacoMonkey/assets/53788631/80867d65-65e2-4152-98c5-e90a2e37c15a)
## 6. **Volcano Mountain**:
![volcano_biome](https://github.com/SIGGgames/PacoMonkey/assets/53788631/7cf34307-9d76-4aa2-a2b9-95486a4c1d71)
![volcano_biome_v2](https://github.com/SIGGgames/PacoMonkey/assets/53788631/6d5fc207-07db-472f-a892-6e66f3cab159)
![volcano_biome_v3](https://github.com/SIGGgames/PacoMonkey/assets/53788631/4ddac3fc-0348-4a0c-8b9d-ccdb5b5db611)

Btw I like this biomes and the path through all of them. Great job 👍

**Reply by julianogales** [reply]
Thanks :)
(The ChatGPT creations are really awesome)

**Comment by julianogales** [comment]
To avoid the feeling of static backgrounds, we could add moving elements and animate them a bit.
> e.g.
> ![e9f1591cd39f7861c549d26ba89f487b](https://github.com/SIGGgames/PacoMonkey/assets/150971528/9827c9f3-421f-47ed-9994-7690d7e3cd6e)
